src/common/855Translator.js

```diff
@@ -48,7 +48,8 @@ export function to855(purchaseOrder, lineItems) {
     text = text.concat(`\\`);
     numberOfSegments++;
   }
-  lineItems.forEach((lineItem, index) => {
+  let index = 0;
+  lineItems.forEach((lineItem) => {
```

Comment:
what's the reason for creating the index variable?

Reply:
The forEach() method on a map doesn't pass the index into the function like the forEach() method on an array so I just pulled it out above

Reply:
Gotcha thanks for explaining! lets start the index at 1 then on line 51, then on line 54 we can do index++ and remove the index++ on line 83